workflow/rocoto/gfs_tasks.py

```diff
@@ -796,6 +796,35 @@ def postsnd(self):
 
         return task
 
+    def fbwinds(self):
+
+        atmos_prod_path = self._template_to_rocoto_cycstring(self._base["COM_ATMOS_GRIB_GRID_TMPL"], {'RUN': self.cdump, 'GRID': '0p25'})
+        deps = []
+        data = f'{atmos_prod_path}/{self.cdump}.t@Hz.pgrb2.0p25.f006'
+        dep_dict = {'type': 'data', 'data': data}
```

Comment:
```suggestion
        dep_dict = {'type': 'data', 'data': data, 'age': 120}
```

If the following patch is added to `rocoto.py`, one can use the `age` attribute as shown above:
```diff
❯❯❯ git diff                                                                                                               develop ✱
diff --git i/workflow/rocoto/rocoto.py w/workflow/rocoto/rocoto.py
index b3f88f01d..c6598c2a4 100644
--- i/workflow/rocoto/rocoto.py
+++ w/workflow/rocoto/rocoto.py
@@ -193,6 +193,7 @@ def _add_data_tag(dep_dict: Dict[str, Any]) -> str:
     dep_type = dep_dict.get('type', None)
     dep_data = dep_dict.get('data', None)
     dep_offset = dep_dict.get('offset', None)
+    dep_age = dep_dict.get('age', None)  # age must be in seconds

     if dep_data is None:
         msg = f'a data value is necessary for {dep_type} dependency'
@@ -206,7 +207,10 @@ def _add_data_tag(dep_dict: Dict[str, Any]) -> str:

     assert len(dep_data) == len(dep_offset)

-    strings = ['<datadep>']
+    if dep_age is None:
+        strings = ['<datadep>']
+    else:
+        strings = [f'<datadep age="{dep_age}">']
     for data, offset in zip(dep_data, dep_offset):
         if '@' in data:
             offset_str = '' if offset in [None, ''] else f' offset="{offset}"'
```

Reply:
``` 
         <dependency>
                <and>
                        <datadep age="120"><cyclestr>&ROTDIR;/gfs.@Y@m@d/@H//products/atmos/grib2/0p25/gfs.t@Hz.pgrb2.0p25.f006</cyclestr></datadep>
                        <datadep age="120"><cyclestr>&ROTDIR;/gfs.@Y@m@d/@H//products/atmos/grib2/0p25/gfs.t@Hz.pgrb2.0p25.f012</cyclestr></datadep>
                        <datadep age="120"><cyclestr>&ROTDIR;/gfs.@Y@m@d/@H//products/atmos/grib2/0p25/gfs.t@Hz.pgrb2.0p25.f024</cyclestr></datadep>
                </and>
        </dependency>
```

Done.